src/commands/app/scaffold.ts

```diff
@@ -35,16 +37,54 @@ export default class AppScaffold extends Kommand {
       },
     ]);
 
-    await tasks.run();
-    this.log('')
-    this.logOk(`Scaffolding complete! Use ${chalk.grey(`cd ${destination} && npm run docker npm install`)} install dependencies and then ${chalk.grey(`npm run docker:dev`)} to run your application!`);
+    try {
+      await tasks.run();
+
+      this.log('')
+      this.logOk(`Scaffolding complete! Install dependencies with :
+        ${chalk.grey(`cd ${destination} && npm run docker npm install`)}
+        and run your application with:
+        ${chalk.grey('npm run docker:dev')}`)
+    }
+    catch (error: any) { }
   }
 
   async cloneTemplate(flavor: string, destination: string) {
-    await execute('git', 'clone', '--depth=1', 'https://github.com/kuzzleio/project-templates', '--branch', flavor, '--single-branch');
+    const templatesDir = '/tmp/kourou-template';
```

Comment:
You should get a tmp dir with the https://www.npmjs.com/package/tmp package instead so it will support Windows